### ISSUE TITLE
Link 404s

### ISSUE THREAD

**Issue by OriginLive** [sotrh/learn-wgpu#403]
Hi, in https://sotrh.github.io/learn-wgpu/intermediate/tutorial10-lighting/#ray-path-tracing the link to raytracing tutorial 404s.

Im very interested in raytracing so it'd be cool if it could be fixed

**Comment by CatCode79** [comment]
The link to me opens it without problems, however I sometimes get a 404 error trying to visit github links. I always thought there was some kind of underlying problem; exactly from which web page you tried to visit the link?

**Comment by OriginLive** [comment]
Hi, from the page I've linked to, there's a link to raytacing tutorial that 404w.

Edit: now it's correctly saying comming soon :/

**Comment by CatCode79** [comment]
IMHO the elusive bug of github 404 struck again, who knows what it is..

With a short search I found:

"It may also be a problem with Github itself. [This page](https://www.githubstatus.com/) tells you the status of Github. I had the same problem, and no fix worked. The status page said "degraded performance", and I just had to wait for the servers to be fully functional again."

Source: https://stackoverflow.com/questions/11577147/how-to-fix-http-404-on-github-pages
